es/index.js

```diff
@@ -175,7 +201,9 @@ export default function axiosRetry(axios, defaultOptions) {
         config.timeout = Math.max(config.timeout - lastRequestDuration, 1);
       }
 
-      return axios(config);
+      return new Promise((resolve) =>
+        setTimeout(() => resolve(axios(config)), retryDelay(currentState.retryCount, error))
```

Comment:
You need to subtract the delay from `config.timeout`, so the global `timeout` value is honored.

Reply:
Thanks, I had missed that.